docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -468,7 +468,7 @@ where t is the period for which transition the is estimated (a year) eg. 365 day
     - 0-59 months old
     - 0.488 (95% CI:0.374-0.604)
     - normal
-    - baseline scenario value. assumed to be the same for SAM and MAM and across age groups.
+    - Baseline scenario value. Assumed to be the same for SAM and MAM and across age groups at the draw level.
```

Comment:
The additional value of treating MAM (and not just SAM) is something I have heard interest in, so we might want/need to extend this to afford different levels of baseline/scaled-up coverage of SAM and MAM in the future.